### DISCUSSION TITLE
dpgen install issue in a docker container

### DISCUSSION THREAD

**Discussion by shuaijiang-ustc** [deepmodeling/dpgen#408] [Q&A]
Hi, I intend to install dpgen in a docker container with non-administrative privileges, so I try to install it with another direcotory instead of ./local/bin, e.g. pip install --prefix /ghome/jiangshuai/dpgen_test
However, even though the installation is successfully with no error reporting, when dpgen -h, it shows: ModuleNotFoundError: No module named 'dpgen'.  Does anyone know how to fix this or is there another way to install dpgen?
Thanks in advance.

**Comment by njzjz** [comment]
I think you need to add the `bin` directory you installed DP-GEN, maybe `/ghome/jiangshuai/dpgen_test/bin`, into `PATH` environment.
